forward-message-mailgun/tests/forward-message-mailgun.test.js

```diff
@@ -0,0 +1,33 @@
+const helpers = require("../../test/test-helper");
+const mailgun = require("mailgun-js");
+const forwardMessageToMailgun = require("../functions/forward-message-mailgun").handler;
+const Twilio = require("twilio");
+
+const context = {
+  MAILGUN_API_KEY: "API_KEY",
+  DOMAIN: "DOMAIN",
+  TO: "test_to@example.com",
+  FROM: "test_from@example.com",
```

Comment:
These environment variables should be `TO_EMAIL_ADDRESS` and `FROM_EMAIL_ADDRESS`.